dev_guide/deployments.adoc

```diff
@@ -99,46 +105,47 @@ spec:
 <4> An xref:image-change-trigger[image change trigger] trigger causes a new replication controller to be
 created each time a new version of the `origin-ruby-sample:latest` image stream tag is available.
 <5> The xref:rolling-strategy[Rolling strategy] is the default way of deploying your pods. May be omitted.
-<6> The existing running deployment will not be paused. Pod template or image changes cover the full spectrum of changes on which deployments currently run.
-<7> Revision history limit is the limit of old replication controllers you want to keep around for rolling back. May be omitted. If omitted, old replication controllers will not be cleaned.
+<6> Pause a deployment configuration. This allows for multiple changes on the pod template before actually rolling it out.
```

Comment:
maybe describe that this will mute the ConfigChange trigger and ImageChange trigger?